cmake/Thirdparty/GetGitRevisionDescription.cmake.in

```diff
@@ -0,0 +1,66 @@
+# Boost Software License - Version 1.0 - August 17th, 2003
```

Comment:
For each new file not under Apache License v2, we need to mention it in this file https://github.com/apache/singa/blob/master/LICENSE#L466.
Pls add the newly added files there. 
Thanks.